test/integration/test_integration.py

```diff
@@ -0,0 +1,168 @@
+from unittest import mock, TestCase
+import argparse
+
+import fhirpipe
+import fhirpipe.cli.run as run
+from fhirpipe.load.fhirstore import get_fhirstore, get_mongo_client
+
+
+### Run from mapping file ###
+args = argparse.Namespace(
+    chunksize=None,
+    config="config-test.yml",
+    mapping="test/fixtures/mimic_mapping.json",
+    multiprocessing=False,
+    reset_store=True,
+    resources=None,
+    source=None,
+)
+
+
+@mock.patch("fhirpipe.cli.run.parse_args", return_value=args)
+def test_run_gql(_):
+    run.run()
+    assert_result_as_expected()
+
+
+"""
+### Run with graphQL queries ###
+args = argparse.Namespace(
+    chunksize=None,
+    config="config-test.yml",
+    mapping="mapping-file.json",
+    multiprocessing=False,
+    reset_store=True,
+    resources=None,
+    source=None,
+)
+
+@mock.patch("fhirpipe.cli.run.parse_args", return_value=args)
+def test_run_from_file(_):
+    run.run()
+"""
+
+### Run for a list of resources ###
+args = argparse.Namespace(
+    chunksize=None,
+    config="config-test.yml",
+    mapping="test/fixtures/mimic_mapping.json",
+    multiprocessing=False,
+    reset_store=True,
+    resources=["Patient", "not_existing_resource"],
+    source=None,
+)
+
+
+@mock.patch("fhirpipe.cli.run.parse_args", return_value=args)
+def test_run_resource(_):
+    run.run()
+
+    mongo_client = get_mongo_client()[fhirpipe.global_config["fhirstore"]["database"]]
+
+    assert mongo_client.list_collection_names() == ["Patient"]
+    assert mongo_client["Patient"].count_documents({}) == 100
+
+    sample = mongo_client["Patient"].find_one({"identifier.0.value": "40655"})
```

Comment:
same remark for the patient identifier (I like to use a variable whenever it is used at more than one place)